Week3/homework/1-step3.js

```diff
@@ -1,11 +1,16 @@
 'use strict';
 
 function foo(func) {
-  // What to do here? 
+
+  console.log('Hello, I am foo! I can call another function:');
+  func();
+
```

Comment:
Nice work! 
I would suggest returning the `func()` call.
The `foo` function does not know how the function will be used, and so should return the results of calling the function.
map, filter, and reduce all return the results of calling the function on the element.